trio/_highlevel_open_tcp_stream.py

```diff
@@ -173,8 +187,12 @@ def format_host_port(host, port):
 #   AF_INET6: "..."}
 # this might be simpler after
 async def open_tcp_stream(
-    host, port, *, happy_eyeballs_delay=DEFAULT_DELAY, local_address=None
-):
+    host: str | bytes,
+    port: int,
+    *,
+    happy_eyeballs_delay: float | None = DEFAULT_DELAY,
+    local_address: str | None = None,
```

Comment:
No clue why the docstring says this must be a str, but since `bind` can take an `Address` this can too.

`Address` defined at
https://github.com/CoolCat467/trio/blob/51347e3772aa54cbc5e1b431c285e01a7a813fc7/trio/_socket.py#L43

Reply:
Fixed annotation in 62b3c9a

Reply:
Ah, digging further into the docstring and https://docs.python.org/3/library/socket.html?highlight=bind#socket-families it looks like it indeed does have to be a string.